kubernetes/linux/Dockerfile.multiarch

```diff
@@ -28,33 +55,84 @@ ENV AZMON_COLLECT_ENV False
+COPY --from=builder /busybin /busybin
+COPY --from=builder /var/opt/microsoft /var/opt/microsoft
+COPY --from=builder /var/lib/logrotate /var/lib/logrotate
+COPY --from=builder /var/spool/cron /var/spool/cron
+
+# executables
+COPY --from=builder /usr/bin/bash /usr/bin/bash
+COPY --from=builder /usr/bin/ruby /usr/bin/ruby
+COPY --from=builder /usr/lib/ruby /usr/lib/ruby
+COPY --from=builder /usr/bin/inotifywait /usr/bin/inotifywait
+COPY --from=builder /usr/sbin/busybox /usr/sbin/busybox
+COPY --from=builder /usr/bin/fluent-bit /usr/bin/fluent-bit
+COPY --from=builder /opt/telegraf /opt/telegraf
+COPY --from=builder /usr/sbin/crond /usr/sbin/crond
+COPY --from=builder /usr/bin/vim /usr/bin/vim
+COPY --from=builder /usr/share/vim /usr/share/vim
+COPY --from=builder /usr/sbin/mdsd /usr/sbin/mdsd
+COPY --from=builder /usr/sbin/logrotate /usr/sbin/logrotate
+COPY --from=builder /usr/sbin/setcap /usr/sbin/setcap
+COPY --from=builder /usr/bin/curl /usr/bin/curl
+COPY --from=builder /usr/bin/jq /usr/bin/jq
+COPY --from=builder /usr/bin/base64 /usr/bin/base64
+COPY --from=builder /usr/bin/fluentd /usr/bin/fluentd 
+
+# bash dependencies
+COPY --from=builder /lib/libreadline.so.8 /lib/
+COPY --from=builder /usr/lib/libncursesw.so.6 /usr/lib/libtinfo.so.6 /usr/lib/
+# inotifywait dependencies
+COPY --from=builder /lib/libinotifytools.so.0 /lib/
+# crond dependencies
+COPY --from=builder /lib/libselinux.so.1 /lib/libpam.so.0 /lib/libc.so.6 /lib/libpcre.so.1 /lib/libaudit.so.1 /lib/libcap-ng.so.0/ /lib/
+# vim dependencies
+COPY --from=builder /lib/libm.so.6 /lib/libtinfo.so.6 /lib/
+# ruby dependencies
+COPY --from=builder /usr/lib/libruby.so.3.1 /usr/lib/libz.so.1 /usr/lib/libgmp.so.10 /usr/lib/libcrypt.so.1 /usr/lib/libm.so.6 /usr/lib/
+# fluent-bit dependencies
+COPY --from=builder /lib/libssl.so.1.1 /lib/libcrypto.so.1.1 /lib/libyaml-0.so.2 /lib/libsystemd.so.0 /lib/libm.so.6 /lib/libgcc_s.so.1 /lib/liblzma.so.5 /lib/liblz4.so.1 /lib/libcap.so.2 /lib/libgcrypt.so.20 /lib/libgpg-error.so.0 /lib/
+# telegraf dependencies
+COPY --from=builder /lib/libc.so.6 /lib/
+# mdsd dependencies
+COPY --from=builder /usr/lib/libdl.so.2 /usr/lib/librt.so.1 /usr/lib/libpthread.so.0 /usr/lib/libm.so.6 /usr/lib/libstdc++.so.6 /usr/lib/libgcc_s.so.1 /usr/lib/
+COPY --from=builder /usr/local/lib/libtcmalloc_minimal.so.4 /usr/local/lib/
+# logrotate dependencies
+COPY --from=builder /lib/libselinux.so.1 /lib/libpopt.so.0 /lib/libpcre.so.1 /lib/
+# curl dependencies
+COPY --from=builder /lib/libcurl.so.4 /lib/libz.so.1 /lib/libnghttp2.so.14 /lib/libssh2.so.1 /lib/libgssapi_krb5.so.2 /lib/libzstd.so.1 /lib/
+COPY --from=builder /usr/lib/libkrb5.so.3 /usr/lib/libk5crypto.so.3 /usr/lib/libcom_err.so.2 /usr/lib/libkrb5support.so.0 /usr/lib/libresolv.so.2 /usr/lib/
+# jq dependencies
+COPY --from=builder /lib/libjq.so.1 /lib/libonig.so.5 /lib/
 
 # Do vulnerability scan in a seperate stage to avoid adding layer
-FROM base_image AS vulnscan
+FROM distroless_image AS vulnscan

```

Comment:
why are running trivy on distroless_image and not on ContainerInsights?

Reply:
> why are running trivy on distroless_image and not on ContainerInsights?

> distroless image is the one which we will be releasing to the customers. why do you think we should do against ContainerInsights?

Reply:
Updated the code. Had to remove a line above. Please look at the changes and let me know if works @pfrcks